docs/css/index_new.scss

```diff
@@ -102,63 +111,109 @@ body {
 }
 
 .HomeContainer {
-  width: 1182px;
+  width: 1199px;
   text-align: center;
   margin: 0 auto;
   font-family: 'fira-sans';
 }
 
-.headline {
-  border: 1px solid #ccc;
-  border-radius: 25px;
-  padding: 18px 30px 19px;
-  display: inline-block;
-  margin-bottom: 120px;
-  font-size: 16px;
-  font-weight: 600;
-  letter-spacing: 0.01em;
-  color: #949494;
-}
-
-.flowContainer {
-  margin-bottom: 122px;
-}
+.FlowContainer {
+  padding-bottom: 120px;
+  position: relative;
+  font-family: 'fira-sans';
+  overflow: hidden;
 
-.imageLeft {
   .imgContainer {
-    float: left;
+    background-repeat: no-repeat;
+    height: 353px;
+    width: 550px;
   }
 
-  .captionContainer {
-    float: left;
-    margin-left: 96px;
+  &#flowContainer1 .imgContainer {
+    background-image: url({{path}}/interim/index1.jpg);
```

Comment:
these should be taken to html as inline styles which will be generated by liquid params.